packages/schemas/src/text/propPanel.ts

```diff
@@ -39,9 +39,12 @@ const UseDynamicFontSize = (props: PropPanelWidgetProps) => {
     changeSchemas([{ key: 'dynamicFontSize', value: val, schemaId: activeSchema.id }]);
   };
   const label = document.createElement('label');
-  label.innerText = i18n('schemas.text.dynamicFontSize') || '';
+  const span = document.createElement('span');
+  span.innerText = i18n('schemas.text.dynamicFontSize') || '';
+  span.style.cssText = 'margin-left: 0.5rem';
```

Comment:
I change this so that the checkbox is on the left side of the label, to match the antd layout that is used for `Required`.

I couldn't find an easy way to use a proper checkbox widget from antd though and make the functionality work though.

Reply:
Nice.